src/MicrosoftTeams.ts

```diff
@@ -576,6 +576,26 @@ export function getUserJoinedTeams(
   callbacks[messageId] = callback;
 }
 
+/**
+ * @private
+ * Hide from docs
+ * ------
+ * Allows an app to get the configuration setting value
+ * @param callback The callback to invoke when the value is retrieved.
+ * @param key The key for the config setting
+ */
+export function getConfigSetting(
```

Comment:
Please sync latest changes from master and move your changes related to getConfigSetting and into ringId ./src/private/privateAPIs.ts file